src/test/java/com/neo4j/docker/TestExtendedConf.java

```diff
@@ -98,7 +105,78 @@ void testReadsTheExtendedConfFile_defaultUser(String password) throws Exception
 		}
 	}
 
+	@ParameterizedTest
+	@ValueSource( strings = {"", "secretN30"} )
+	void testReadsExistingConfFileWithExtension_rootUser( String password ) throws Exception
+	{
+		// set up test folders
+		Path testOutputFolder = HostFileSystemOperations.createTempFolder( "extendedConfIsRead-" );
+		Path logsFolder = HostFileSystemOperations.createTempFolder( "logs-", testOutputFolder );
 
+		try ( GenericContainer container = createContainer( password ) )
+		{
+			HostFileSystemOperations.mountHostFolderAsVolume( container, logsFolder, "/logs" );
+			container.start();
+			Container.ExecResult result = container.execInContainer(
+					"bash", "-c", "printf '\\ndbms.logs.http.rotation.keep_number=$(expr 2 * 10)\\n' >> ${NEO4J_HOME}/conf/neo4j.conf"
+			);
+			String logs = String.format( "stdout:\n%s\n\nstderr:\n%s", result.getStdout(), result.getStderr() );
+			Assertions.assertEquals( 0, result.getExitCode(), logs );
+			restartNeo4j( container );
+
+			runContainerAndVerify( container, null, logsFolder, password );
```

Comment:
it's the same container.
It's only started twice. The `container.start()` in `runContainerAndVerify` is a no-op because it's already running at that point